src/pkg/utils/utils.go

```diff
@@ -138,3 +139,48 @@ func ToLocalFile(t any, filePath string) error {
 func IsRemotePkg(pkg types.Package) bool {
 	return pkg.Repository != ""
 }
+
+func hasScheme(s string) bool {
+	return strings.Contains(s, "://")
+}
+
+func hasDomain(s string) bool {
+	return strings.Contains(s, ".") && len(s) > 1
```

Comment:
May need to add a bit more logic here, `-o .tmp` is not a domain but rather a hidden file